### ISSUE TITLE
Items rendering either after window resize or by pressing f12

### ISSUE THREAD

**Issue by Himanshupal0001** [namespace-ee/react-calendar-timeline#871]
**Describe the bug**

**To Reproduce**

_Please provide one or both of the following:_

Code (If you have a branch somewhere that shows your issue) or link to codesandbox example. [Here's a base template for this library](https://codesandbox.io/s/w6xvqzno4w):

OR

**Expected behavior**

**Screenshots**

**Library Version**
Please provide version of library in which this bug is occurring.

**Desktop (please complete the following information):**

* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

**Smartphone (please complete the following information):**

* Device: [e.g. iPhone6]
* OS: [e.g. iOS8.1]
* Browser [e.g. stock browser, safari]
* Version [e.g. 22]

**Additional context**

**Comment by Himanshupal0001** [comment]
### Here's before and after pictures
**Before**
![image](https://user-images.githubusercontent.com/49358057/187409754-fd84674f-136b-4dea-9249-c481dc6cb5c5.png)


**After**
![image](https://user-images.githubusercontent.com/49358057/187409776-fb871477-8b2f-403d-9e7e-b3ea2bacc9ae.png)

**Comment by clemencestr** [comment]
I have the same problem on Mozilla, but it does not appear on Google Chrome or Safari.

**Comment by Himanshupal0001** [comment]
> I have the same problem on Mozilla, but it does not appear on Google Chrome or Safari.

I tried in all the browsers and the issue still persist.

**Comment by Himanshupal0001** [comment]
> I can reproduce this by passing a React state variable to the groups. However, if I pass a normal array then it works as expected.

Yeah, If A normal array is passed it works fine {if the value provided already rather fetched from the backend}. Do you have any idea how to resolve this?  I tried normal arrays rather than states but the issue still persists.

**Comment by baatten** [comment]
what does a normal array mean? I have this bug even with demo example. anyone know how to resolve it?

**Comment by fsuk** [comment]
After spending an afternoon with this issue I have found that the issue occurrs when groups/items is `undefined` such as you might get if using useState before it is set.

To fix this I changed my code to check if groups and items are set before rendering:

```
 {groups && items &&     
        <Timeline
          groups={groups}
          items={items}
          ...
        />
      }
```

More detail:
This seems to be a bug in the ScrollElement with the` scrollComponent.scrollLeft` being 0 initially until after the Timeline is resized.

**Comment by PieterT2000** [comment]
@fsuk is correct, only render the timeline component if groups are defined
